test/extended/util/annotate/generated/timeout/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- deads2k
+- soltysh
+
+options:
+  no_parent_owners: true
```

Comment:
@deads2k this will ensure that only the approvers from this file can approve changes to this directory, which is what you wanted.

Reply:
> @deads2k this will ensure that only the approvers from this file can approve changes to this directory, which is what you wanted.

Yeah, this approach looks good.  Do you have a general reviewer to watch these?  I might suggest @stbenjam.  He'll probably need some background, which I'd suggest placing in a comment at the top of the generated file or in a readme.md.